BUILDING.md

```diff
@@ -138,6 +138,9 @@ meson compile -C _mesonbuild
 
 The compiled binary should now be here: `./_mesonbuild/target/release/rnote`.
 
+Note that if an older version of rnote has previously been installed, the old `gschema` file will still be used, which
```

Comment:
I'd add  a short explanation about what the gschema file is/does, mentioning that the issues can arise when it has been modified after when the build was last  installed. And explicitely mention the error you encountered, so it is easier to make the connection between the error and this note